azure-pipelines.yml

```diff
@@ -9,10 +9,16 @@
 #
 # SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
 
-
 trigger: [ '*' ]
 pr: [ '*' ]
 
+variables:
+  BUILD_PIPELINE_ID: "$(System.DefinitionId)"
+  BUILD_ID: "$(Build.BuildId)"
+  ${{ if startsWith(variables['Build.SourceBranch'], 'refs/heads/') }}:
+    GIT_BRANCH: "$(Build.SourceBranch)"
+  ${{ if startsWith(variables['Build.SourceBranch'], 'refs/pull/') }}:
```

Comment:
Maybe something similar would make it possible for "forked" CI runs to clone from the forked copy and use the same branch. That would help especially with C++ builds when something is done that touches both repositories.

Clearly this comment is not a request for making a change to this PR 🙂